fill/fill_station.cc

```diff
@@ -153,9 +174,22 @@ class RocketTelemeterServiceImpl final : public RocketTelemeter::Service
                     // Broken stream
```

Comment:
similar watchdog here I think except you should pet before writing since this one checks the rocket connection